frontend/rpm/template.go

```diff
@@ -105,6 +108,11 @@ func (w *specWrapper) Requires() fmt.Stringer {
 		writeDep(b, "BuildRequires", name, constraints)
 	}
 
+	if len(w.Artifacts.Services) > 0 {
+		// We take advantage of the systemd-rpm-macros package to simplify the service file installation
+		writeDep(b, "BuildRequires", "systemd-rpm-macros", nil)
```

Comment:
This can't really be set here since it is distro specific (and probably not right to have in the spec itself).
I think this could be added to the base builder image directly.

Reply:
This should be fixed now.